### DISCUSSION TITLE
console창에 표기되는 오류 처리 질문

### DISCUSSION THREAD

**Discussion by HyeongtaekOh** [wafflestudio/seminar-2021#567] [Q&A] [answered]
오류처리를 하면 console창에 다음과 같이 표기되는 것도 없어져야되는 것인가요?
![스크린샷, 2021-10-08 18-48-06](https://user-images.githubusercontent.com/86342741/136535727-c81e2d3f-925b-46aa-b6a9-5f5505e7ce72.png)

**Comment by woohm402** [comment, marked as answer]
아뇨 괜찮습니다! 아마 오류처리 아무리 해도 안 없어질 거예요

얘만 안뜨면 됩니다!
![image](https://user-images.githubusercontent.com/39977696/136538838-8e48b298-e2a7-41ca-ab50-9048b04e3f82.png)

**Reply by HyeongtaekOh** [reply]
감사합니다!
